cmd/boot-script-service/oauth.go

```diff
@@ -259,3 +251,38 @@ func JWTIsValid(jwtStr string) (jwtValid bool, reason, err error) {
 
 	return
 }
+
+func (client *OAuthClient) FetchAccessToken(remoteUrl string) (string, error) {
```

Comment:
Could we add a descriptive comment for this function?

Reply:
Resolved in [e8d32b6](https://github.com/OpenCHAMI/bss/pull/26/commits/e8d32b693a5b1e9b8b85f28f3cf32e818d31256e).